src/Lazy.elm

```diff
@@ -0,0 +1,202 @@
+module Lazy exposing
```

Comment:
This code seems to be copy-pasted from [lazy-tree-with-zipper](https://package.elm-lang.org/packages/turboMaCk/lazy-tree-with-zipper/3.1.1/). What was your reasoning behind copy-pasting the code (as opposed to depending on that package in the elm.json)?

Reply:
@jhrcek Additional functions needed to be implemented: https://github.com/GlobalWebIndex/challenge-elm/pull/14/files#diff-c7375ff11bc41bd2c1fbde6f98f5904aR35